src/modules/ens/sagas.ts

```diff
@@ -27,13 +27,26 @@ import {
   FetchENSListRequestAction,
   fetchENSListRequest,
   fetchENSListSuccess,
-  fetchENSListFailure
+  fetchENSListFailure,
+  SET_ALIAS_REQUEST,
+  SetAliasRequestAction,
+  setAliasSuccess,
+  setAliasFailure
 } from './actions'
 import { ENS, ENSOrigin, ENSError } from './types'
 import { getLands } from 'modules/land/selectors'
 import { FETCH_LANDS_SUCCESS } from 'modules/land/actions'
+import { CatalystClient, DeploymentBuilder, DeploymentWithMetadataContentAndPointers } from 'dcl-catalyst-client'
+import { EntityType } from 'dcl-catalyst-commons'
+import { env } from 'decentraland-commons'
+import { Avatar } from 'decentraland-ui'
+import { Personal } from 'web3x-es/personal'
```

Comment:
These non-relative imports should be on top :) 🔝

Reply:
Fixed! thanks @cazala